test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,25 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty
+        assertIsNotNull();
+        assertIsNotNull("");
+        
+        // no null
+        assertIsNotNull("xxx");
+        assertIsNotNull("123");
+        assertIsNotNull("yyy", "qwer123");
+        assertIsNotNull("xxx", "xyz123", "asd");
+        assertIsNotNull("qwerty", 123, "hello", "hi");
+
+        // all null or mix of null with objects
+        assertIsNull((Object) null);
+        assertIsNull(null, null);
+        assertIsNull("test", null);
+        assertIsNull(null, "abc", null);
```

Comment:
Very comprehensive set of tests! However, do consider the case where the object is a list, what if there are null objects within the list?